charts/sentry/templates/relay/_helper-sentry-relay.tpl

```diff
@@ -74,8 +74,27 @@ config.yml: |-
     {{- else }}
     redis: "{{ $redisProto }}://{{ $redisHost }}:{{ $redisPort }}/{{ $redisDb }}"
     {{- end }}
+
+    {{- if ((.Values.kafkaTopicOverrides).prefix) }}
+    topics:
+      metrics_sessions: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-metrics"
+      events: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-attachments"
+      transactions: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-transactions"
+      outcomes: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-outcomes"
```

Comment:
Could you tell me, what is this topic `ingest-outcomes`? I don't see the provisioning of this topic [here](https://github.com/sentry-kubernetes/charts/blob/develop/charts/sentry/values.yaml#L2004-L2106).

Reply:
My bad, it should be `outcomes. I’ll correct.